### PR TITLE
Feature venv

### LINKED ISSUE #77: provide an automatic virtualenv setup script
this makes the setup process much easier, when you don't want / can't use vagrant.

### PR DESCRIPTION
**Pull request by sedrubal**
- Fixes #77 
  - Not an automatic install script but quite detailed information how to do it.
- using `requirements.txt` by default

### CODE REVIEW

**Comment by mgmax** [comment]
if I follow the virtualenv instructions, does it still work as described or do I have to activate the virtualenv every time when starting FabLabKasse?

**Comment by sedrubal** [comment]
You have to be in the virtual environment if you installed the dependencies (only) inside the environment: `. .env/bin/activate && ./run.py`

File: INSTALLING [hunk 1]
```diff
@@ -69,6 +64,20 @@ Modem-Manager interferes with the serial port. It is highly recommended to remov
 
  TODO
 
+### 1c. Virtualenv
```
Comment (mgmax) [inline INSTALLING]: it should be "1e" instead of 1c. Then feel free to merge.

Comment (sedrubal) [inline INSTALLING]: lol danke. Fixed with 3b6abd523994a303e5550dbbd09b517a9ccc06e0

**Pull request merged**